_posts/2022-11-02-relicensing.md

```diff
@@ -26,8 +53,10 @@ In this post we want to share the motivation, outline the relicensing
 process, and invite questions from the community.
 
 
+## Rationale for license change
```

Comment:
Added top level section here so we can link to it, also makes it cleaner organisation-wise (I _think_?)